pkg/kubelet/network/exec/exec_test.go

```diff
@@ -177,3 +177,28 @@ func TestPluginTearDownHook(t *testing.T) {
 		t.Errorf("Mismatch in expected output for teardown hook. Expected '%s', got '%s'", expectedOutput, string(output))
 	}
 }
+
+func TestPluginStatusHook(t *testing.T) {
+	pluginName := fmt.Sprintf("test%d", rand.Intn(1000))
+	defer tearDownPlugin(pluginName)
+	installPluginUnderTest(t, "", pluginName)
+
+	plug, err := network.InitNetworkPlugin(ProbeNetworkPlugins(testPluginPath), pluginName, network.NewFakeHost(nil))
+
+	ip, err := plug.Status("namespace", "name", "dockerid2345")
```

Comment:
We really ought to test the plugin returning "" as well as error cases.  This just got a lot more complicated to test